pages/api/randomProfile.js

```diff
@@ -0,0 +1,40 @@
+// pages/api/randomProfile.js
```

Comment:
Please move filter to `api/discover/random.js`

Reply:
Done!